pkg/csi/service/common/common_controller_helper.go

```diff
@@ -30,6 +30,14 @@ import (
 	"sigs.k8s.io/vsphere-csi-driver/v2/pkg/csi/service/logger"
 )
 
+type InvalidConfigError struct {
```

Comment:
does this mean invalid secret config or vc inventory config?
can we provide doc for this

Reply:
Changed the name and provided doc strings to it.